source/dubregistry/dbcontroller.d

```diff
@@ -161,6 +161,21 @@ class DbController {
 		m_packages.update(["name": packname], ["$set": ["categories": categories]]);
 	}
 
+	void setPackageDonationDetails(string packname, string url, string detail)
+	{
+		m_packages.update(["name": packname], ["$set": ["donation_url": url, "donation_detail": detail]]);
```

Comment:
Why couldn't this be part of the `dub.sdl`?

Reply:
huh thats  actually a really good idea, you wont need to sync github or anything then